docs_user/main.adoc

```diff
@@ -6,6 +6,8 @@
 :doctype: book
 :context: osp
 
+include::../docs_user/rhoso_attributes.adoc[]
```

Comment:
We're navigating to the parent directory and then back to `docs_user`. I think this could be just `include::rhoso_attributes.adoc[]`? Unless there's a reason to have it like this because of some downstream/upstream compatibility/likeness.

Reply:
I changed it to what you suggested and it worked without errors (both in the file and the local preview). I'll commit and push my changes.